config/production.json

```diff
@@ -3,5 +3,7 @@
     "max_static_paths": 9999,
     "revalidate": 360
   },
-  "flags": {}
+  "flags": {
+    "enable_new_downloads_view": true
```

Comment:
This would show the new view on our assembly branch I believe since `HASHI_ENV` is set to `production`. Is this what we want? (Totally fine if yes, just double checking)

Reply:
Woops, copy/paste error. Forgot to update this one to `false`! 😆

Reply:
Fixed in https://github.com/hashicorp/dev-portal/pull/126/commits/ab82fc18cab422d5091364ad9f9574db78745899